huaweicloud/resource_huaweicloud_obs_bucket.go

```diff
@@ -85,6 +85,12 @@ func resourceObsBucket() *schema.Resource {
 				},
 			},
 
+			"quota": {
```

Comment:
please add the new `quota` to the documents as well.

Reply:
@niuzhenguo done.